SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -857,6 +865,44 @@ FString FSpatialGDKEditorToolbarModule::GetOptionalExposedRuntimeIP() const
 	}
 }
 
+void FSpatialGDKEditorToolbarModule::UpdateMobileClient() const
```

Comment:
Should we be specific and make this `UpdateIOSClient` or something since this is mac only.

Reply:
ah yes, that should be `UpdateIOSClient`, however it's not Mac only. you can use it on a windows machine as well